app/models/like.rb

```diff
@@ -1,2 +1,12 @@
 class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_save :updatepostcounter
```

Comment:
- Kudos for using one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save 🚀💯⭐
-